sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClientIntegrationTest.java

```diff
@@ -160,8 +159,7 @@ public void beginUpdatePhoneNumberCapabilities(HttpClient httpClient) {
                 return result.getFinalResult();
             })
         ).assertNext((PurchasedPhoneNumber acquiredPhoneNumber) -> {
-            assertEquals(PhoneNumberCapabilityType.INBOUND_OUTBOUND, acquiredPhoneNumber.getCapabilities().getSms());
-            assertEquals(PhoneNumberCapabilityType.INBOUND, acquiredPhoneNumber.getCapabilities().getCalling());
+            assertNotNull(acquiredPhoneNumber);
```

Comment:
I thought we decided to get the current capabilties and switch to different the capabilities each time?

Reply:
Example for Python: https://github.com/Azure/azure-sdk-for-python/pull/18386/files